src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -163,6 +128,24 @@ public void linkDefendant(String authorisation) {
             ).forEach(caseId -> linkToCase(defendantUser, anonymousCaseWorker, letterHolderId, caseId)));
     }
 
+    private Optional<Claim> getCaseBy(String input, String authorisation, ImmutableMap<String, String> searchString) {
```

Comment:
Please remove `input` since it duplicates value of `ImmutableMap<String, String> searchString`. Also no point to pass argument just for logging purpose. Either print search criteria in logs or case ID retrieved from data store.